edx_rbac/utils.py

```diff
@@ -63,6 +63,8 @@ def user_has_access_via_database(user, role_name, role_assignment_class, context
     The context argument is evaluated against the context(s) received from the role_assignment_class while accounting
     for the ALL_ACCESS_CONTEXT to grant access.
     """
+    if user.is_anonymous:
```

Comment:
Perhaps `if hasattr(user, 'is_anonymous') and user.is_anonymous:`, in case the `user` argument passed in is `None`?

Reply:
...or `getattr(user, 'is_anonymous', False)`, if we're doing code golf :)

Reply:
That, too haha. Basically, just want to be sure we prevent an error from occurring when trying to access a non-existent attribute on a `None` type 😄

Reply:
✅